src/shared/components/unknownStudies/UnknownStudiesWarning.tsx

```diff
@@ -1,26 +1,70 @@
 import * as React from 'react';
 import { observer } from 'mobx-react';
+import { Collapse } from 'react-collapse';
+import classnames from 'classnames';
 
 @observer
 export default class UnknownStudiesWarning extends React.Component<
     { ids: String[] },
+    { studiesCollapsed: boolean },
     {}
 > {
+    constructor(props: { ids: String[] }) {
+        super(props);
+        this.state = {
```

Comment:
we try to use mobx state instead of react state.  for consistency.  just put it as a property on the component